### PR TITLE
Clarify dbt setup steps

### PR DESCRIPTION
**Pull request by dot2dotseurat**
Steps below are not relevant, this is a setup update. 

I've checked that:

* [ ] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [ ] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] the directory tree matches the pattern /sector/blockchain/ e.g. /tokens/ethereum
* [ ] if adding a new model, I added a test
* [ ] the filename is unique and ends with .sql
* [ ] each file has only one view, table or function defined  
* [ ] column names are `lowercase_snake_cased`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,9 +4,24 @@
...
+```
 
-Enter `.` or other placeholders after running `dbt init`. This will not connect to the database but you have access to some dbt actions. When you are prompted to choose a target, please enter `wizard` so we know you are an external contributor.
+If the env is created successfully, enter the env shell. (If you run into Python version errors, try installing the matching version or editing the python version in the Pipfile but don't commit your change).
+```
+pipenv shell
+```
+
+Set up the dbt spellbook project. (You must run this step from the spellbook directory)
+Select Databricks and enter `.` or other placeholders after running `dbt init` for the host, HTTP path, and token. This will not connect to the database but you have access to some dbt actions.
```
Comment (soispoke) [inline README.md:19]: nit: Maybe rephrase this the the dbt init comes first in the sentence

**Review by soispoke — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (dot2dotseurat) [reply README.md]: yes, thats a great point. I'll update.
